docker-compose.yaml

```diff
@@ -77,6 +77,7 @@ services:
       - MONGO_CONNECTION_STRING=mongodb://mongo:mongo@mongo:27017
       - MONGO_DATABASE_NAME=kakarot-local
       - MAX_FELTS_IN_CALLDATA=30000
+      - RETRY_TX_INTERVAL=0.5
```

Comment:
this won't work will it? since you use a usize

Reply:
True, just put 1 sec to simplify